cpp/src/arrow/acero/aggregate_node_test.cc

```diff
@@ -210,5 +210,36 @@ TEST(GroupByNode, NoSkipNulls) {
   AssertExecBatchesEqualIgnoringOrder(out_schema, {expected_batch}, out_batches.batches);
 }
 
+TEST(GroupByNode, AnyAndOr) {
```

Comment:
```suggestion
TEST(ScalarAggregateNode, AnyAll) {
```

Reply:
This is for `ScalarAggregateNode` rather than `GroupByNode`. And it's for `any` and `all`, not `or`.